src/main/native/windows/file.cc

```diff
@@ -781,5 +781,46 @@ bool GetCwd(std::wstring* result, DWORD* err_code) {
   }
 }
 
+std::wstring GetCorrectCasing(const std::wstring& abs_path, bool with_unc) {
```

Comment:
this looks like an absolute path is required - I don't see this requirement on the Java side. Do I miss something?

Reply:
currently the only caller passes with_unc=false, right? Do you have plans to change that?

Reply:
Removed.